packages/plugin-lighthouse/src/lib/lighthouse-plugin.ts

```diff
@@ -1,25 +1,31 @@
-import { AuditOutputs, PluginConfig } from '@code-pushup/models';
+import { Audit, AuditOutputs, PluginConfig } from '@code-pushup/models';
+import { filterByAuditSlug, filterBySlug } from '@code-pushup/utils';
 import { AUDITS, GROUPS, LIGHTHOUSE_PLUGIN_SLUG } from './constants';
 
+type AuditSlug = (typeof AUDITS)[number]['slug'];
 export type LighthousePluginOptions = {
   url: string;
   outputPath?: string;
-  onlyAudits?: string | string[];
+  onlyAudits?: AuditSlug | AuditSlug[];
```

Comment:
This doesn't work, unfortunately, `AuditSlug` is the same as `string` here. The `meta.id` from Lighthouse is a `string`, so we're not going to get a more specific type than that.